cpp/sopt/proximal.h

```diff
@@ -11,9 +11,7 @@
 #include "sopt/mpi/utilities.h"
 #endif
 
-namespace sopt {
-//! Holds some standard proximals
-namespace proximal {
+namespace sopt::proximal {
 
```

Comment:
The comment here doesn't seem very useful but if they put it in I think we should keep it for the documentation.

Reply:
Yes. Agreed. Will fix and update the PR. Thank you.

Reply:
Fixed. Thanks a lot for pointing this out.